test/pstl_offload/memory/usm_memory_replacement.pass.cpp

```diff
@@ -11,6 +11,7 @@
 #error "PSTL offload compiler mode should be enabled to run this test"
 #endif
 
+#define _CRT_SECURE_NO_WARNINGS 1
```

Comment:
Let's make if under `#ifdef` if it's Windows only

Reply:
Done.